### PR TITLE
test-errors

### PR DESCRIPTION
**Pull request by kyhungtran191**

### CODE REVIEW

**Review by minhan1910 — COMMENTED** [review]
bad

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class UserController extends Controller
+{
+    //
+    public function __construct()
+    {
+
+    }
+    public function showErrors()
+    {
+        return 'Con xin lỗi chú';
```
Comment (minhan1910) [inline app/Http/Controllers/UserController.php]: ????? what the hell ??

**Review by minhan1910 — COMMENTED** [review]
Review showErrors at UserController file

File: app/Http/Controllers/UserController.php (hunk 1 above)
Comment (kyhung19102) [reply app/Http/Controllers/UserController.php:16]: con cac dap chet me may gio thang leader l

File: app/Http/Controllers/UserController.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+class UserController extends Controller
+{
+    //
+    public function __construct()
+    {
+
+    }
+    public function showErrors()
+    {
+        return 'Con xin lỗi chú Quách Phúc';
```
Comment (minhan1910) [inline app/Http/Controllers/UserController.php:16]: OKi nha 💯

**Review by minhan1910 — COMMENTED** [review]
Good dog

**Pull request closed without merging**